metrics/prometheus.go

```diff
@@ -409,3 +419,9 @@ func (p *Prometheus) MeasureBackend5xx(start time.Time) {
 func (p *Prometheus) IncErrorsStreaming(routeID string) {
 	p.proxyStreamingErrorsM.WithLabelValues(routeID).Inc()
 }
+
+func (p *Prometheus) ObserveRouteKubeResources(ingress string, routeId string, service string) {
```

Comment:
this could be shortened into:
```suggestion
func (p *Prometheus) ObserveRouteKubeResources(ingress, routeId, service string) {
```